src/app/applications/datatarget/fiware/fiware-detail/fiware-detail.component.html

```diff
@@ -31,6 +31,24 @@ <h3>{{ 'DATATARGET.DETAILS' | translate }}</h3>
               <p>{{ 'DATATARGET.NO-AUTHORIZATIONHEADER' | translate }}</p>
             </ng-template>
 
+            <p><strong>{{ 'DATATARGET.TOKENENDPOINT' | translate }}</strong></p>
+            <code><pre *ngIf="datatarget.tokenEndpoint; else showNoTokenEndpointText">{{datatarget.tokenEndpoint}}</pre></code>
+            <ng-template #showNoTokenEndpointText>
+              <p>{{ 'DATATARGET.NO-TOKENENDPOINT' | translate }}</p>
+            </ng-template>
+
+            <p><strong>{{ 'DATATARGET.CLIENTID' | translate }}</strong></p>
+            <code><pre *ngIf="datatarget.clientId; else showNoClientIdText">{{datatarget.clientId}}</pre></code>
+            <ng-template #showNoClientIdText>
+              <p>{{ 'DATATARGET.NO-CLIENTID' | translate }}</p>
+            </ng-template>
+
+            <p><strong>{{ 'DATATARGET.CLIENTSECRET' | translate }}</strong></p>
+            <code><pre *ngIf="datatarget.clientSecret; else showNoClientSecretText">{{''.padStart(20, '*')}}</pre></code>
```

Comment:
Functions inside HTML are bad for performance. Set it in the component and reference it